lib/modules/datasource/helm/schema.ts

```diff
@@ -0,0 +1,82 @@
+import { z } from 'zod';
+import { detectPlatform } from '../../../util/common';
+import { parseGitUrl } from '../../../util/git/url';
+import { regEx } from '../../../util/regex';
+import { LooseRecord } from '../../../util/schema-utils';
+import type { Release } from '../types';
+
+const HelmReleaseSchema = z.object({
+  version: z.string(),
+  created: z.string().nullable().catch(null),
+  digest: z.string().optional().catch(undefined),
+  home: z.string().optional().catch(undefined),
+  sources: z.array(z.string()).catch([]),
+  urls: z.array(z.string()).catch([]),
```

Comment:
so catch ensures it still works if one of those is missing?

Reply:
Yes, exactly